fmatch/matcher.py

```diff
@@ -34,7 +34,7 @@ def __init__(
         self.es = Elasticsearch([self.es_url], timeout=30, verify_certs=verify_certs)
         self.data = None
 
-    def get_metadata_by_uuid(self, uuid: str, index: str = None) -> dict:
+    def get_metadata_by_uuid(self, uuid: str, index: str = None, uuid_field: str="uuid") -> dict:
```

Comment:
Is it worth it to extract uuid string literal as a constant?

Reply:
updated to set in init to not have to pass to each function, thoughts?